lang/en.json

```
@@ -102,7 +158,8 @@
   "DG.Roll.Target": "Target: ",
   "DG.Roll.Success": "Success",
   "DG.Roll.Failure": "Failure",
-  "DG.Roll.Critical": "Critical",
+  "DG.Roll.CriticalSuccess": "¡Critical Success!",
+  "DG.Roll.CriticalFailure": "¡Critical Failure!",
```

in English the leading exclamation point is not included. :)
```suggestion
  "DG.Roll.CriticalSuccess": "Critical Success!",
  "DG.Roll.CriticalFailure": "Critical Failure!",
```